src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DhGaugeParameters.hpp

```diff
@@ -0,0 +1,132 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <string>
+
+#include "Options/Options.hpp"
+
+namespace GeneralizedHarmonic::gauges {
+/*!
+ *  A struct holding the parameters for initializing damped harmonic gauge
+ *  If UseRollon is true, the gauge transitions ("rolls on") to     damped
+ * harmonic fromdetermines whether the gauge transitions to damped harmonic
+ * ("rolls on") from the initial data gauge; otherwise, the gauge begins
+ * immediately in damped harmonic gauge.
+ */
+template <bool UseRollon>
+struct DhGaugeParameters;
+
+template <>
```

Comment:
Since the two specializations aren't documented, can you `/// \cond` them so they don't show up in Doxygen?